app/services/streaming/streaming.ts

```diff
@@ -88,13 +88,15 @@ export class StreamingService extends StatefulService<IStreamingServiceState>
   /**
    * @deprecated Use toggleStreaming instead
    */
+  // TODO: no usages anymore, can we remove, including on interface?
```

Comment:
This method is called via our socket API by some 3rd party applications.  Alex is working on separating out this API to make this more obvious, but we should remove these comments.